### DISCUSSION TITLE
Refused to apply style...

### DISCUSSION THREAD

**Discussion by euri10** [jesec/flood#296] [Q&A]
On chrome only, if I try to reach: `https://flood.domain.com` I get this in the console

```
Refused to apply style from 'https://flood.domain.com/static/css/main.64ffd5346316ac349de3.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
````

In Firefox I get redirected to `https://flood.domain.com/login`

I wondered if it was my config or something that should be dealt here.

**Comment by jesec** [comment]
I can not reproduce the issue with Chromium 92. 

Are you running Flood with reserve proxy? 

If you do, make sure that you configured the web server correctly.

**Reply by euri10** [reply]
> I can not reproduce the issue with Chromium 92.
> 
> Are you running Flood with reserve proxy?
> 
> If you do, make sure that you configured the web server correctly.

I just use traefik yes, no webserver, and now you mention it I kind of remember traefik does not serve static file, that may explain it.
I'm using  Chrome Version 89.0.4389.90 (Official Build) (64-bit)
THis is just weird the redirect happens in Firefox but not in chrome

**Comment by euri10** [comment]
I used ctrl+F5, this solved the issue.
The error message happened after i upgraded the container, maybe a cache issue in the browser, not sure.
